Content.Server/Chemistry/TileReactions/CreateEntityReaction.cs

```diff
@@ -54,7 +58,8 @@ public FixedPoint2 TileReact(TileRef tile, ReagentPrototype reagent, FixedPoint2
                 }
             }
 
-            entMan.SpawnEntity(Entity, tile.GridPosition().Offset(new Vector2(0.5f, 0.5f)));
+            var newEnt = entMan.SpawnEntity(Entity, tile.GridPosition().Offset(new Vector2(0.5f, 0.5f)));
+            newEnt.RandomOffset(RandomOffsetMax);
```

Comment:
Just spawn it at a random offset rather than moving it immediately after.

Reply:
Makes code uglier (can't use RandomOffset now), but will do.